### ISSUE TITLE
Release 32 of the safe-core-sdk doesn't exist on npm

### ISSUE THREAD

**Issue by IvanKodak** [safe-global/safe-core-sdk#605]
I want to use the 1.4.1 version of GnosisCore in my project, but I can't do that because the latest version of the safe-core-sdk on npm doesn't support the 1.4.1 contract version. When will you upload R32 on npm?

The Latest version of the package on npm is 3.3.5, it doesn't support 1.4.1.
![image](https://github.com/safe-global/safe-core-sdk/assets/41439233/034448fc-160b-4ce1-af5c-17d1673a8cb2)

https://www.npmjs.com/package/@safe-global/safe-core-sdk/v/3.3.5?activeTab=versions

**Comment by dasanra** [comment]
Hey @IvanKodak

from mono-repo release `r28` there has been package name changes. `@safe-global/safe-core-sdk` is now published as `@safe-global/protocol-kit`. The `protocol-kit` is compatible with 1.4.1 contracts from `@safe-global/protocol-kit v1.1.0`, that correspond to mono-repo release `r31`.

There is a [migration guide available](https://docs.safe.global/safe-core-aa-sdk/protocol-kit/reference/v1) that will help you migrate to `@safe-global/protocol-kit v1.3.0` directly from `@safe-global/safe-core-sdk`.

We recently published also `@safe-global/protocol-kit v2.0.0`, which has a migration guide available once you are using `@safe-global/protocol-kit v.1.3.0`. The major change in this one is that now it's compatible with `ethers v6` instead `ethers v5`. I recommend you migrate in order so it's easier to follow the guides.

To be more clear on this, we have now marked `@safe-global/safe-core-sdk` as deprecated on npm, adding the migration link to the warning message.
